app/controllers/todos_controller.rb

```diff
@@ -6,39 +6,48 @@ class TodosController < ApplicationController
 
     # GET /todo_lists/:todos_list_id/todos
   def index
+    authorize @todo_list, :show?
+
     todos = @todo_list.todos
     render json: todos, status: :ok
   end
 
     # POST /todo_list/:todo_list_id/todos
   def create
     todo = @todo_list.todos.build(todo_params)
+
+    authorize todo
+
```

Comment:
authorize after building?

Reply:
as authorize method need object to see the permissions here todo is the object or context  where the policy is applied